app/views/V2App/app.scala.html

```diff
@@ -32,7 +32,25 @@
 <body>
     <div class="wrap" id="react-mount"></div>
     <div id="config" data-value="@clientConfigJson"></div>
-    <script src="@jsLocation" type="application/javascript"></script>
+    <script type="module">window.global = window</script>
+
+    @if(isDev) {
+        <!-- Connect directly to Vite dev server in development -->
+        <!-- See https://vitejs.dev/guide/backend-integration.html -->
+        <script type="module">
+            import RefreshRuntime from 'http://localhost:5173/assets/fronts-client/@@react-refresh'
```

Comment:
I'm not familiar with this yet, and I still have many changes to look at so this question may resolve itself but in the meantime: `fronts-client` isn't `fronts-client-v2` here?

Reply:
👍  this just happens to be the URL we serve V2 assets from – but since moving the `dist` assets directory from `fronts-client` to `fronts-client-v2`, this is now confusing! e9ef7f3f704c57f30fd157797dde39b462fb54a4 ensures we're using fronts-client-v2 everywhere assets are needed.